pkg/api/handlers/compat/events.go

```diff
@@ -93,6 +93,10 @@ func GetEvents(w http.ResponseWriter, r *http.Request) {
 			if evt == nil {
 				continue
 			}
+			if evt.Status == events.HealthStatus && !evt.IsHealthStatusChanged{
```

Comment:
This needs a !utils.IsLibpodRequest(r) check like the other docker compat changes below as this function is used for both the docker and libpod endpoint.